src/lib/gossip.ts

```diff
@@ -109,13 +116,6 @@ export class Gossip {
     this.updateGossip();
   }
 
-  incrementStench(): void {
```

Comment:
IMO we should keep the chatbot work silent for the time being, and rely on the whiteboard. When everything is in place to no longer rely on the whiteboard at all, then we can clean up all the unneeded code.

For testing, we have two clans available, so we could have whiteboard folks in one clan, and chat folks in another clan.

Reply:
Suggesting using a pref to enable using clan chat? 

I could update this change to be nearly silent. Do need to write occasionally to ensure we don't enter away mode, which delays chat messages. Update would be to implement your original suggestion of waiting for the current cycle to finish. This way no need to communicate current stench level

Reply:
Note I do believe this change is backwards compatible - should work to have a mix of folks running this new version and older versions. Do agree that when we remove the mutex, everyone will need to update their script as I don't believe that change is backwards compatible

Reply:
brought this function back